scene/2d/canvas_item.cpp

```diff
@@ -94,6 +94,7 @@ void CanvasItemMaterial::_update_shader() {
 		case BLEND_MODE_SUB: code += "blend_sub"; break;
 		case BLEND_MODE_MUL: code += "blend_mul"; break;
 		case BLEND_MODE_PREMULT_ALPHA: code += "blend_premul_alpha"; break;
+		case BLEND_MODE_NO: code += "blend_no"; break;
```

Comment:
Seems ok to me, but I would rename to BLEND_MODE_DISABLED

Reply:
Done